sdcm/utils/decorators.py

```diff
@@ -237,6 +237,10 @@ def wrapped(*args, **kwargs):  # noqa: PLR0914
             result["hdr_summary"] = tester.get_cs_range_histogram(stress_operation=workload,
                                                                   start_time=start,
                                                                   end_time=end)
+            hdr_throughput = 0
+            for summary, values in result["hdr_summary"].items():
+                hdr_throughput += values["throughput"]
+            result["cycle_hdr_throughput"] = round(hdr_throughput)
```

Comment:
where is it used?

Reply:
`hdr_summary` isn't sent to argus at all, so this might be used in email template.
but yes in this PR it's not used, but would be saved into ES

Reply:
> `hdr_summary` isn't sent to argus at all, so this might be used in email template. but yes in this PR it's not used, but would be saved into ES

this is exactly the reason. I'll use it in the gradual test email